sections/05_treat.md

```diff
@@ -78,6 +78,53 @@ able to significantly outperform a random forest classifier trained on gene
 expression changes. These approaches suggest promising directions to improve
 the efficiency of clinical trials and accelerate drug development.
 
+##### Drug repositioning
+
+Drug repositioning (or repurposing) is an attractive option for delivering new
+drugs to the market because of the high costs and failure rates associated with
+more traditional drug discovery approaches [@doi:10.1016/j.jhealeco.2016.01.012
+@doi:10.1038/nrd4609]. Ever since the concept of the Connectivity Map was
+proposed [@doi:10.1126/science.1132939], several advanced computational methods
+have been applied to formulate and validate drug repositioning hypotheses
+[@doi:10.1093/bib/bbv020 @doi:10.1093/bib/bbw112 @doi:10.1093/bib/bbw110]. Using
+supervised learning and recommender systems to tackle this type of problems is
+proving successful in different scenarios, especially when coupled with
+topological information from protein - protein or protein - compound interaction
+networks [@doi:10.1186/1758-2946-5-30 @doi:10.1021/ci500340n
+@doi:10.1186/s12859-015-0845-0]. Menden et al
```

Comment:
If I understand correctly, the `Menden et al` phrase indicates a transition from an overview of methods to those that specifically use neural networks.  We may want to be more explicit about that transition, perhaps even starting a new paragraph.